### ISSUE TITLE
Flipping F and f not Working on Google search results

### ISSUE THREAD

**Issue by Zackhardtoname** [philc/vimium#3628]
**Describe the bug**
```
map F LinkHints.activateModeToOpenInNewForegroundTab 
map f LinkHints.activateMode
```

It works on many sites to flip F and f, but not on Google search results

**To Reproduce**
1. Apply the settings above
2. Go to [google.com](google.com), and search for something
3. press "F," and then the "hint key" for any website 

**Expected**
Open in the current tab

**Actual**
Open in a new tab

**Comment by gdh1995** [comment]
The modes of LinkHints just means Vimium will provide such suggestions to web pages, and a page can apply its rules to decide whether to open in a new tab or not.

**Comment by Zackhardtoname** [comment]
Is there a way to force it?

**Comment by gdh1995** [comment]
No, because a page can always call `preventDefault` on Vimium's simulated events, and then open its wanted URL in a tab using JavaScript
